user_guide/intro_nav_user_menu.rst

```diff
@@ -0,0 +1,58 @@
+.. _user-guide-getting-started-user-menu:
+
+
+User Menu 
+=========
+
+On the right side of the screen under your username you will find the User Menu drop-down. It provides a fast way to 
+access your user profile, calendar, mailbox and task list. It is also used to log out of the system.
+
+
+.. image:: ./img/intro/user_menu.png
+
+My User
+-------
+
+This link opens your user profile that is described in the 
+:ref:`corresponding section <user-guide-getting-started-my-user-overview>` of the 
+:ref:`Log In, View Your User Page, and Set Up Your Profile guide <user-guide-intro-log-in-edit-profile>`.
+
+My Emails
+---------
+
+This link opens the My Emails page—your OroCRM mailbox that stores all the email conversations with your customers or 
+contacts. 
+
+
+*"My Emails"* page contains all the emails:
+
+- Sent to you in OroCRM
+- Sent by you in OroCRM
+- Collected from the external mailboxes. This may be the mailbox for which IMAP connection has 
+  been enabled (if any), as described in the section above or other mailboxes connected to your account. To collect all 
+  the newest emails, click the :guilabel:`Sync Emails` button.
```

Comment:
Edit: Collected from the external mailboxes. This may be the mailbox for which an IMAP connection has been enabled (if any), as described in the section above, or where other mailboxes are connected to your account. To collect all the newest emails, click the :guilabel:`Sync Emails` button.